frontend/testing/requests/highlights.test.js

```
@@ -1,13 +1,28 @@
 const {
   getHighlights,
   getHighlightsByDroplet,
+  getHighlightsForLesson,
+  createHighlight,
+  deleteHighlight,
 } = require("../../lib/requests/highlights");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)